components/sync15/src/request.rs

```diff
@@ -162,6 +172,7 @@ pub struct PostQueue<Post, OnResponse> {
     last_modified: ServerTimestamp,
 }
 
+// How to deal with a batch
```

Comment:
This comment doesn't help me understand anything.

What does a type that implements `BatchPoster` do?